docs-go/flows.md

```diff
@@ -1,33 +1,40 @@
 # Flows
 
-Flows are functions with some additional characteristics: they are strongly
-typed, streamable, locally and remotely callable, and fully observable.
-Firebase Genkit provides CLI and developer UI tooling for working with flows
-(running, debugging).
+Flows are runnable objects with some additional characteristics over simple
```

Comment:
I believe the Go folks have previously instructed me to avoid using the term "object" in Go contexts. I'm not sure what a good replacement is here though. @ianlancetaylor

Reply:
Food for thought: `Flows are wrapped functions with some additional characteristics over direct calls: ...`

In the future, we can supplement that list with "interruptable/resumable" or similar.

Reply:
"wrapped functions" SGTM